src/plugins/data/public/index_patterns/expressions/load_index_pattern.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { ExpressionFunctionDefinition } from '../../../../../plugins/expressions/public';
+import { getIndexPatterns } from '../../services';
+import { IndexPatternSpec } from '../../../common/index_patterns';
+
+const name = 'indexPattern';
```

Comment:
did you mean to name the function `indexPattern` or `indexPatternLoad`?

if the plan is to eventually add other functionality to this function, `indexPattern` makes sense. but if we are only going to use this for loading an index pattern, i'd vote for `indexPatternLoad`

Reply:
in the future i think we want to add something like `indexPatternCreate` which will have all the spec arguments and will allow creating index pattern on-the-fly (we need the static class for that to work)